network-api/networkapi/mozfest/templates/mozfest/mozfest-base.html

```diff
@@ -19,6 +19,13 @@
     {% blocktrans %}Mozilla Festival{% endblocktrans %}{% if page.specifics.seo_title %} - {{ page.specifics.seo_title }}{% else %} - {{ page.title }}{% endif %}
 {% endblock %}
 
+{% block org_schema %}
+  {% if page.structured_data %}
+    <script type="application/json">
```

Comment:
When looking at the code snippet I have for the structured data - the script type is: 
`<script type="application/ld+json">`

Reply:
I think this is because the `ld+` was removed, I've added it back